db/migrate_cards/data/facebook_meta.json

```diff
@@ -0,0 +1,106 @@
+{
+  "url": null,
+  "timestamp": "2015-06-08 17:08:08 +0100",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[facebook meta+*right+*structure]]\n[[Metric+facebook meta description+*type plus right+*structure]]\n[[Article+facebook meta description+*type plus right+*structure]]\n[[User+facebook meta description+*type plus right+*structure]]\n[[Topic+facebook meta description+*type plus right+*structure]]\n[[Company+facebook meta description+*type plus right+*structure]]\n[[Company+facebook meta image+*type plus right+*structure]]\n[[Topic+facebook meta image+*type plus right+*structure]]\n[[User+facebook meta image+*type plus right+*structure]]\n[[Article+facebook meta image+*type plus right+*structure]]\n[[Metric+facebook meta image+*type plus right+*structure]]\n[[Metric+facebook meta title+*type plus right+*structure]]\n[[User+facebook meta type+*type plus right+*structure]]\n[[Article+facebook meta type+*type plus right+*structure]]\n[[facebook meta image+*right+*structure]]\n[[facebook meta title+*right+*structure]]\n[[facebook meta description+*right+*structure]]\n[[facebook meta type+*right+*structure]]\n[[wikirate layout]]",
+    "value": [
+      {
+        "name": "facebook meta+*right+*structure",
+        "type": "HTML",
+        "content": "<div style=\"display:none\">\r\n  <meta property=\"og:site_name\"          content=\"WikiRate.org\" />\r\n  <meta property=\"og:url\"                content=\"http://wikirate.org/{{_l|linkname}}\" />\r\n  <meta property=\"og:type\"               content=\"{{_l+facebook_meta_type|core}}\" />\r\n  <meta property=\"og:title\"              content=\"{{_l+facebook_meta_title|core}}\" />\r\n  <meta property=\"og:description\"        content=\"{{_l+facebook_meta_description|core}}\" />\r\n  <meta property=\"og:image\"              content=\"http://wikirate.org{{_l+facebook_meta_image|core}}\" />\r\n</div>"
+      },
```

Comment:
As I understand it, these meta tags only really matter when facebook itself is making the request.  Therefore, I was thinking perhaps we should only go through the process of rendering all these cards when we know that facebook is the requester.

We could still use the above cards, but we would probably need to add code to identify the facebook agent.